packages/block-editor/src/layouts/flex.js

```diff
@@ -325,6 +325,7 @@ function FlexLayoutJustifyContentControl( {
 
 	return (
 		<ToggleGroupControl
+			__next40pxDefaultSize
```

Comment:
This can be seen for example in the Row block.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/fb6971e2-9d22-4367-8f69-7c1d13084fe7" alt="Flex layout controls, before" width="275">|<img src="https://github.com/user-attachments/assets/faebbf72-2da1-4904-a3c1-7410be8d37d6" alt="Flex layout controls, after" width="275">|